analyzers/src/SonarAnalyzer.CSharp/Rules/UninvokedEventDeclaration.cs

```diff
@@ -38,62 +38,55 @@ public sealed class UninvokedEventDeclaration : SonarDiagnosticAnalyzer
         internal const string DiagnosticId = "S3264";
         private const string MessageFormat = "Remove the unused event '{0}' or invoke it.";
 
-        private static readonly Accessibility maxAccessibility = Accessibility.Public;
+        private static readonly Accessibility MaxAccessibility = Accessibility.Public;
 
-        private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
+        private static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
 
-        private static readonly ISet<SyntaxKind> eventSyntax = new HashSet<SyntaxKind>
+        private static readonly ISet<SyntaxKind> EventSyntax = new HashSet<SyntaxKind>
         {
             SyntaxKind.EventFieldDeclaration,
         };
 
-        protected override void Initialize(SonarAnalysisContext context)
-        {
+        protected override void Initialize(SonarAnalysisContext context) =>
             context.RegisterSymbolAction(RaiseOnUninvokedEventDeclaration, SymbolKind.NamedType);
-        }
 
         private void RaiseOnUninvokedEventDeclaration(SymbolAnalysisContext context)
         {
             var namedType = (INamedTypeSymbol)context.Symbol;
-            if (!namedType.IsClassOrStruct() ||
-                namedType.ContainingType != null)
+            if (!namedType.IsClassOrStruct() || namedType.ContainingType != null)
             {
                 return;
             }
 
             var removableDeclarationCollector = new CSharpRemovableDeclarationCollector(namedType, context.Compilation);
 
             var removableEventFields = removableDeclarationCollector
-                .GetRemovableFieldLikeDeclarations(eventSyntax, maxAccessibility)
+                .GetRemovableFieldLikeDeclarations(EventSyntax, MaxAccessibility)
                 .ToList();
 
             if (!removableEventFields.Any())
             {
                 return;
             }
 
-            var invokedSymbols = GetInvokedEventSymbols(removableDeclarationCollector);
```

Comment:
we'd need to come up with a reproducer, but I suspect this change lead to the #4498 perf regression